crap-advisor-frontend/src/core/components/Map/index.jsx

```diff
@@ -2,34 +2,32 @@ import React, {useEffect, useState} from "react";
 import "./index.css"
 import {MapContainer, TileLayer, ZoomControl} from "react-leaflet";
 import RestroomMarker from "../RestroomMarker";
+import {apiUrl} from "../../../app/constants";
 
-const Map = () => {
-    const apiUrl = "http://localhost:8082"
+const Map = ({selectRestroom}) => {
     const defaultPosition = [53.21176, 50.18394]
     const defaultZoom = 13
     const [restrooms, setRestrooms] = useState([])
 
     useEffect(() => {
         fetch(apiUrl + "/restroom/all")
```

Comment:
You should wrap your api response in try catch construction to avoid unexpectable errors

Reply:
I really should